README.md

```diff
@@ -1,11 +1,203 @@
+
+The result value is of type `FixedPeriod`:
+
+```ts
+type FixedPeriod = {
+    id: string
+    iso?: string
+    name: string
+    startDate: string
+    endDate: string
+}
+```
+
+-   `id` (and `iso`) are the same right now, and return a period identifier, i.e. `2015Q1` (quarter one of the year 2015) or `2015SunW1` (the first week - starting Sunday - of the year 2015). The full list of values are documented in [periodValues](https://github.com/dhis2/multi-calendar-dates/blob/main/features/fixed-periods.ethiopic.feature) in the feature files.
+
+> please use `id` to identify the period. `iso` was added for backwards compatibility with existing implementations
+
+-   `startDate` and `endDate` are the start and end dates of the specific period. This is provided for convenience, but the backend currently only makes use of the ID to work out what each period means.
+
+-   `name` is the human readable name of the period, according to what we dispaly in DHIS2 Data Visualizer, i.e. `Week 1 - 2014-13-03 - 2015-01-04` for a weekly period or `2015` for a yearly period.
+
+## getNowInCalendar
+
+`getNowInCalendar` returns today's date in the specified calendrical system.
+
+### Examples
+
+```
+// Assuming today's date is 13th October 2021
+beforeEach(() => {
+    // 13 October 2021 UTC
+    jest.spyOn(Date, 'now').mockReturnValue(1634089600000)
+})
+it('should get today date in Gregorian', () => {
+    const { day, month, year } = getNowInCalendar('gregory', 'UTC')
+    expect({ day, month, year }).toEqual({ day: 13, month: 10, year: 2021 })
+})
+it('should get today date in Ethiopic', () => {
+    const {
+        day,
+        month,
+        eraYear: year,
+    } = getNowInCalendar('ethiopic', 'UTC')
+    expect({ day, month, year }).toEqual({
+        day: 3,
+        month: 2,
+        year: 2014,
+    })
+})
+```
+
+### Types
+
+The method takes two arguments:
+
+-   `calendarToUse` which can be one of the calendars specified [here](https://github.com/dhis2/multi-calendar-dates/blob/multi-calendar-docs/src/constants/calendars.ts) (defaults to `gregory` calendar if not specied.
+-   `timeZone`: a string representing the time zone (defaults to `UTC`) of the user.
+
+and returns a [ZonedDateTime](https://tc39.es/proposal-temporal/docs/zoneddatetime.html) object, which can be destructured to `.year` or `.eraYear` (`eraYear` preferred to avoid an issue with Ethiopic calendar), `.month`, `.day` which returns the values in the specified calendar, or you can `.getISOFields()` to return the underlying iso8601 (gregory) date.
```

Comment:
```suggestion
and returns a [ZonedDateTime](https://tc39.es/proposal-temporal/docs/zoneddatetime.html#properties) object, which can be destructured to `.year` or `.eraYear` (`eraYear` preferred to avoid an issue with Ethiopic calendar), `.month`, `.day` which returns the values in the specified calendar, or you can `.getISOFields()` to return the underlying iso8601 (gregory) date.
```

I realized this section on the page might be more helpful for readers